ingest_python_scripts/launch_ingest.py

```
@@ -0,0 +1,203 @@
+         "--timeout-seconds", "20",
+         "--parameters", params,
+         "--query", "Command.CommandId",
+         "--output", "text"],
+        capture_output=True, text=True,
+    )
+    if r.returncode != 0:
+        return ""
+    cmd_id = r.stdout.strip()
+    for _ in range(8):
+        time.sleep(3)
+        status = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)